### PR TITLE
MM-54013- Removing some mattermost references

### PR DESCRIPTION
**Pull request by asaadmahmood**

### CODE REVIEW

**Review by esethna — APPROVED** [review]
Thanks Asaad!

**Comment by asaadmahmood** [comment]
@sbishel Can you accept this PR, this is just focalboard website related stuff.

File: website/site/layouts/partials/footer.html [hunk 1]
```diff
@@ -180,6 +180,23 @@
             </li>
         </ul>
     </div>
+</footer>  */}}
```
Comment (sbishel) [inline website/site/layouts/partials/footer.html:183]: Is there a reason this commented code is being retained?

**Review by sbishel — CHANGES_REQUESTED** [review]
Just some commented code that needs deleted. Otherwise, LGTM

File: website/site/layouts/partials/footer.html (hunk 1 above)
Comment (asaadmahmood) [reply website/site/layouts/partials/footer.html]: @sbishel Removed it.

**Review by sbishel — APPROVED** [review]

**Pull request merged**